tests/test_barostat.py

```diff
@@ -185,12 +182,24 @@ def test_barostat_is_deterministic():
         seed,
     )
 
-    ctxt = custom_ops.Context(coords, v_0, host_box, integrator_impl, u_impls, barostat=baro)
+    ctxt = custom_ops.Context(coords, v_0, host_box, integrator.impl(), u_impls, barostat=baro)
     ctxt.multiple_steps(15)
     atm_box = ctxt.get_box()
     # Verify that the volume of the box has changed
     assert compute_box_volume(atm_box) != compute_box_volume(host_box)
-    np.testing.assert_almost_equal(compute_box_volume(atm_box), box_vol, decimal=5)
+
+    baro = custom_ops.MonteCarloBarostat(
+        coords.shape[0],
+        pressure,
+        temperature,
+        group_indices,
+        barostat_interval,
+        u_impls,
+        seed,
+    )
+    ctxt = custom_ops.Context(coords, v_0, host_box, integrator.impl(), u_impls, barostat=baro)
+    ctxt.multiple_steps(15)
+    assert compute_box_volume(atm_box) == compute_box_volume(ctxt.get_box())
```

Comment:
maybe a comment here saying what this test is doing? (eg we get bitwise reproducible boxes?)

Reply:
Added comment: https://github.com/proteneer/timemachine/pull/1023/commits/6a80a10cbfec03dc37a7655bf121914aef702df2